src/libstd/f32.rs

```diff
@@ -970,6 +970,7 @@ impl f32 {
     /// assert!((-3.0f32).clamp(-2.0f32, 1.0f32) == -2.0f32);
     /// assert!((0.0f32).clamp(-2.0f32, 1.0f32) == 0.0f32);
     /// assert!((2.0f32).clamp(-2.0f32, 1.0f32) == 1.0f32);
+    /// assert!((std::f32::NAN).clamp(-2.0f32, 1.0f32).is_nan());
     /// ```
```

Comment:
I think we may use assert_eq macro for those assertions.

Reply:
For the ones other than the new one, I agree, but because that's existing stuff, I don't think we need to block this PR landing for that.